torchacc/ops/flash_attn.py

```diff
@@ -165,24 +165,23 @@ def backward(ctx, dout, *args):
 class FlashAttnVarlenXla(torch.autograd.Function):
 
     @staticmethod
-    def forward(ctx, q, k, v, cu_seqlens_q, cu_seqlens_k, max_seqlen_q,
-                max_seqlen_k, dropout_p, softmax_scale, causal, window_size,
-                alibi_slopes, deterministic, return_softmax):
+    def forward(ctx, q, k, v, attention_mask, dropout_p, softmax_scale, causal,
+                window_size, alibi_slopes, deterministic, return_softmax):
         if softmax_scale is None:
             softmax_scale = q.shape[-1]**(-0.5)
         assert isinstance(window_size, tuple) and len(window_size) == 2
 
-        softmax_lse, out, rng_state = torch_xla._XLAC._flash_attention_forward(
```

Comment:
`SPMDFlashAttnVarlenXla` and `FlashAttnVarlenQKVPackedXla` also need to be updated.

Reply:
Done. But I found no UT for them.